app_internal_test.go

```diff
@@ -46,6 +46,29 @@ func TestAppRun(t *testing.T) {
 	wg.Wait()
 }
 
+// TestAppRunStart tries to trigger a startup error when running an app and not exit(1).
+func TestAppRunStart(t *testing.T) {
+	// Setting exit code to be successful.
+	app := New(Invoke(struct{}{}), ExitCode(0))
```

Comment:
If deleting this option causes the test to fail, that means that the test binary is being exited. We definitely do not want to exit the test binary.